docs/pages/next_major_development.md

```diff
@@ -20,6 +20,39 @@ The current tasks are listed here: [Integrate new PMD logo #1931](https://github
 
 The new logo is available from the [Logo Project Page](pmd_projectdocs_logo.html).
 
+### Revamped Command Line Interface
+
+PMD now ships with a unified Command Line Interface for both Linux/Unix and Windows. Under the `pmd` executable (`pmd.bat` for Windows),
+all utilities can be launched similar to how git is normally used. All commands and options are thoroughly documented in the help,
```

Comment:
I think this reference to git comes a bit out of nowhere. I'd suggest to rephrase to make it clear what the delta is

```suggestion
PMD now ships with a unified Command Line Interface for both Linux/Unix and Windows. Instead of having a collection of scripts 
for the different utilities shipped with PMD, a single script `pmd` (`pmd.bat` for Windows) can now launch all
utilities using subcommands, e.g. `pmd run`, `pmd designer`. All commands and options are thoroughly documented in the help,
```